### PR TITLE
feature/FOUR-12430: Redesign Fields

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
Redesign like the mockup defined in Figma:
image-20231116-180723

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-12430

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-5d8984b620.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-5d8984b620.eng.processmaker.net

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-5d8984b620.eng.processmaker.net

File: resources/js/processes/scripts/components/CreateScriptModal.vue [hunk 1]
```diff
@@ -9,22 +9,19 @@
     >
       <i class="fas fa-plus" /> {{ $t("Script") }}
     </b-button>
-    <modal
+    <b-modal
```
Comment (fagubla) [inline resources/js/processes/scripts/components/CreateScriptModal.vue:12]: don't change this tag

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/processes/scripts/components/CreateScriptModal.vue (hunk 1 above)
Comment (luNunezProcessmaker) [reply resources/js/processes/scripts/components/CreateScriptModal.vue:12]: without the tag it wouldn't let me change the footer

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-5d8984b620.eng.processmaker.net

**Review by fagubla — APPROVED** [review]

File: resources/js/processes/scripts/components/CreateScriptModal.vue [hunk 2]
```diff
@@ -395,8 +427,26 @@ export default {
...
+  }
+  .form-group-border{
+    border-bottom: 1px solid #CDDDEE;
+  }
   .collapsed > .opened,
   :not(.collapsed) > .closed {
       display: none;
   }
+  .script-form {
+    overflow-y: scroll;
```
Comment (fagubla) [inline resources/js/processes/scripts/components/CreateScriptModal.vue:447]: use auto instead of scroll

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-5d8984b620.eng.processmaker.net

**Review by fagubla — APPROVED** [review]

**Pull request merged**